src/virtual-select.js

```diff
@@ -659,6 +659,7 @@ export class VirtualSelect {
         this.onGroupTitleClick($option);
       } else {
         this.selectOption($option, { event: e });
+        $option.focus();
```

Comment:
@joselrio, this line should have a comment explaining why this focus in here

Reply:
Forgot to remove it! Done!